### PR TITLE
Updating the rh-1.0 OCP checks

### PR DESCRIPTION
**Pull request by KiranBodipi**
1. Added audit commands wherever required.
2. Updated the scripts with type to manual if title contains Manual.
3. Updated the scripts with test_items wherever required.
4. Fixed a typo.

### CODE REVIEW

File: cfg/rh-1.0/policies.yaml [hunk 1]
```diff
@@ -212,7 +227,9 @@ groups:
 
       - id: 5.3.2
         text: "Ensure that all Namespaces have Network Policies defined (Manual)"
-        type: "manual"
```
Comment (deven0t) [inline cfg/rh-1.0/policies.yaml:215]: I think we should have this as manual, otherwis the command output will be very big. and we dont have test so there is no point to run this command.

**Review by deven0t — APPROVED** [review]
LGTM

**Comment by KiranBodipi** [comment]
Hey @chen-keinan can you please help to review and merge this PR

**Review by mjshastha — APPROVED** [review]
I'm content with the changes.

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**